### PR TITLE
Test/postings

### LINKED ISSUE #18: Additional UI for position selection
As a user, when I click on an open position, the color should change indicating selected(or something similar). And the color should stay when I visit this event detail page again.

### LINKED ISSUE #27: upcoming jobs redirect to job details displaying page
As a user, I should be able to click on my upcoming jobs and be redirected to the event details.

- [x] Click on my upcoming jobs and be redirected to the event details
- [x] Restyling

### LINKED ISSUE #29: Display category on single event page view
As a user, I should see event's category when review details.

### LINKED ISSUE #31: bug/style-sidebar-resizing
On resizing the window vertically, 

the sidebar squeezes all elements inside.

We need to have a fixed position for the sidebar.

### LINKED ISSUE #33: bug/style-open-positions-bars
We need to fix the responsiveness of the open positions elements. 

Issue:
On resizing the browser, the text falls out the element block. 

Possible Fix:
create media queries or adjust text size on resizing of the element

### LINKED ISSUE #34: single-event-page/change-submit-button
Issue:

Right now from the use perspective it is a little confusing what SUBMIT button does on click.

Possible Fix:
Change the name of the button
Provide a small instruction/description what  the user can do on this event page

### LINKED ISSUE #36: sidebar/hide-button-hover
As the user

I want to see the change in style when I hover over the "hide sidebar" button on the sidebar

### LINKED ISSUE #37: single-event-page/go-back-button-hover
As the user,

I would like to the change in style when I hover over or click on 'go back' button on the single event page

### PR DESCRIPTION
**Pull request by coopterrones**
#### What's this PR do?
-Styling updates 
-Testing user view on homepage
-Testing postings on homepage
Add mixins to the styles
Add style to the filter input and fixes in main tag
add initial postings test
Remove underline for link
Display and style category in event details
Update color for upcoming job card
test event postings on homepage
testing posting card css rules
Merge branch 'feature/css-styling-fixes' into test/postings
Add media query and fix resize issue on event page
Remove outline and add hover for user-toggle-button
test search input functionality in homepage
test sort button functionality on homepage
Add mixins to refactor stykes
Fix user sidebar styles
Fix small responsive styling issues
Add effect to a chosen position on a single event page
Change submit name button
Test the filter by categories in Navigation component
test the title of homepage view
test a click on an open posting should redirect to a new url
Merge branch 'feature/css-styling-fixes' into test/postings
#### What does it fix?
-small css bugs / updates

#### Where should the reviewer start?
-`postings-test` line 1
-`user-test` line 1

#### How should this be manually tested?
-test that all tests pass and css styling updates do not creates any bugs

### CODE REVIEW

File: cypress/integration/Postings_test.js [hunk 1]
```diff
@@ -81,5 +81,11 @@ describe("Testing Postings component", () => {
     cy.get(".posting-detail").should("not.contain", "Food Devlivery");
   });
 
-  
+  it("should direct to a new URL when a posting card is clicked", () => {
+    cy.get(".posting-wrapper")
+      .contains("Something Crazy")
```
Comment (okalimbet) [inline cypress/integration/Postings_test.js:86]: I learned a new way test! I didnt know you can create such a big chain!

**Review by okalimbet — APPROVED** [review]
We need to decide on a consistency of test syntax. In some files we break chains and in this file we keep them all in one line.

File: src/Components/PostingView/index.js [hunk 2]
```diff
@@ -121,6 +122,12 @@ const PostingView = ({ match, getUserInfo }) => {
               </strong>
               <p>{organization}</p>
             </div>
+            <div className="posting-category-wrapper">
```
Comment (okalimbet) [inline src/Components/PostingView/index.js:130]: Nice catch ! I didnt even notice we dont have it displayed!

**Review by okalimbet — APPROVED** [review]

File: cypress/integration/Postings_test.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import _mockData from "./_ mockData";
+
+describe("Testing Postings component", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/");
+
+    cy.intercept(
+      {
```
Comment (yiranli624) [inline cypress/integration/Postings_test.js:8]: Great way stubbing a response! Good work!

**Review by yiranli624 — APPROVED** [review]

**Pull request merged**